sql/core/src/test/scala/org/apache/spark/sql/jdbc/JDBCV2Suite.scala

```diff
@@ -344,8 +344,7 @@ class JDBCV2Suite extends QueryTest with SharedSparkSession with ExplainSuiteHel
       case _: DataSourceV2ScanRelation =>
         val expected_plan_fragment =
           "PushedAggregates: [MAX(ID), MIN(ID)], " +
-            "PushedFilters: [IsNotNull(ID), GreaterThan(ID,0)], " +
-            "PushedGroupby: []"
```

Comment:
don't we print `PushedAggregates` and `PushedGroupby` together in `DataSourceScanExec.scala`? Why remove `PushedGroupby` only here?

Reply:
The query is `select MAX(ID), MIN(ID) FROM h2.test.people where id > 0`. The aggregates `MAX(ID)` and `MIN(ID)` are pushed down. There is no group by, so pushed group by is [].

Reply:
Yes I understand the PushedGroupby is empty here. I don't know why we need to remove it here. We can still test it to be empty, right?

Reply:
OK. I see what you mean now.